lib/Doctrine/DBAL/Platforms/DB2Platform.php

```diff
@@ -878,4 +881,57 @@ protected function getReservedKeywordsClass()
     {
         return Keywords\DB2Keywords::class;
     }
+    
+    /**
+     * Prepare ORDER BY string for OVER() if applicable
+     * 
+     * @param  string $query
+     * 
+     * @return string
+     */
+    private function getOrderByForOver($query)
```

Comment:
Since the DBAL requires PHP ^7.1, we should use scalar type hints and return types where applicable.

Reply:
I was hoping someone would say that!
Based on the current code, without checking composer.json, I wasn't sure if I should apply that.